load-test/artillery/verify_api_key.yaml

```diff
@@ -17,8 +17,21 @@ config:
   http:
     timeout: 10
   phases:
-    - duration: 10
+    - duration: 5
+      arrivalRate: 100
+      name: "arrival rate=100"
+    - pause: 5
+    - duration: 5
+      arrivalRate: 200
+      name: "arrival rate=200"
+    - pause: 5
+    - duration: 5
+      arrivalRate: 300
+      name: "arrival rate=300"
+    - pause: 5
+    - duration: 5
```

Comment:
Can you also add for 1000? This gives us the behavior for a wide range of data set. Might help catch bugs that affect performance with greater sensitivity.

Reply:
500, 600... 1000, or just 1000?